.github/workflows/build.yaml

```diff
@@ -0,0 +1,42 @@
+name: "Build and deploy frontend"
+on:
+  push:
+    branches:
+      - "**"
+      - "!main"
+      - "!master"
+    paths:
+      - '**'
+      - '.github/workflows/*.yml'
+
+jobs:
+  deploy:
+    name: Build and deploy frontend manually
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@67fbcbb121271f7775d2e7715933280b06314838
+        with:
+          role-to-assume: ${{ secrets.IAM_ROLE }}
+          aws-region: ${{ secrets.AWS_REGION }}
+      - name: Set Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - name: Install dependencies
+        run: |
+          yarn install  
```

Comment:
@ruggerocastagnola con @acolella92 ci chiedevamo se fosse strettamente necessario utilizzare npm, in questo caso verrà eliminato lo yarn.lock e inizializzato il progetto con npm.

Reply:
Sì usiamo `npm` per favore, eliminiamo lo `yarn.lock` e lasciamo che sia committato il `package-lock.json`